### PR TITLE
new-path for the code reviewer

### PR DESCRIPTION
**Pull request by aradradev**
**A form for the website E-learning**
- Added some html and css 
- Added the modern form and checkbox
- Added a style and a linter to check the code

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <div class="main-content">
     <h1 id="title">freeCodeCamp Survey Form</h1>
     <p id="description">Thank you for taking the time to help us improve the platform</p>
-    
+       
         <div class="name">
             <label for="name" id="name-label">Name</label>
             <input type="text" id="name" name="name" placeholder="Enter your name" required>
```
Comment (aradradev) [inline index.html:17]: Please you have to wrap your formular like your inputs and checkbox inside a form tag. Beacause don't using a form is not required in the HTML5. Here is a link on how to use a form: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form

File: index.css [hunk 2]
```diff
@@ -24,9 +24,7 @@ body{
     margin: 8px 0;
 }
 
-#survey-form{
-    width: 50%;
-    
+#survey-form{ width: 50%;
     margin: 0 auto;
```
Comment (aradradev) [inline index.css:28]: I noticed in the css file there is an error in the line 27, for more information please reade this article for best practice HTML&CSS https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md

File: index.css
Comment (aradradev) [inline index.css:1]: There is a linter errors in your CSS 😄 so, please check your code again and remove the errors untile you stop seeying the errors 👍  .

**Review by aradradev — COMMENTED** [review]
**Your html file is not structred as a HTML5, so you have to wrap your code with a form ♻️ **

File: index.css
Comment (iambenkis) [reply index.css:1]: Keep doing the great work 👍

**Review by iambenkis — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- Why of comments .:heavy_check_mark:
-  Inline comments :heavy_check_mark: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @iambenkis in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**